src/styles/index.scss

```diff
@@ -1,3 +1,121 @@
 body {
   margin: 0;
+  background-color: rgb(116, 116, 116);
```

Comment:
Consider using SCSS variables for main values like `background-color`, `font-family`, etc. This helps in maintaining consistency and ease of updates.